cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/orion/OrionBackend.java

```diff
@@ -111,7 +122,23 @@ JsonResponse getSubscriptionsByIdV2(String token, String subscriptionId, String
      * @throws Exception
      * @return response
      */
-    JsonResponse getSubscriptionsV2(String token, String subscriptionId, String fiwareService, 
-            String fiwareServicePath) throws Exception;
-    
+    JsonResponse getSubscriptionsV2(String token, String subscriptionId, String fiwareService, String fiwareServicePath)
+            throws Exception;
+
+    /**
+     * Create or update entity in the system (v2).
```

Comment:
I understand that "v2" referst to "NGSIv2". I would use the second one (to avoid be misunderstood, e.g. the second version of the updateRemoveContext() function).